ViewModelKit/VMKTableViewRowActionCreator.m

```diff
@@ -21,13 +21,48 @@ - (instancetype)initWithTableView:(UITableView *)tableView rowActionsType:(id<VM
     return self;
 }
 
-- (nullable NSArray<UITableViewRowAction *> *)tableViewRowActions {
+#pragma mark - UISwipeActionsConfiguration
+
+- (nullable UISwipeActionsConfiguration *)swipeActionsConfiguration {
+    Class swipeContainer = NSClassFromString(@"_UITableViewCellSwipeContainerView");
+    NSMutableArray<UIContextualAction *> * actions = [NSMutableArray array];
+    for (VMKTableViewRowActionViewModel *rowAction in self.rowActionsType.rowActions) {
+        UIContextualAction *tableViewAction = [UIContextualAction contextualActionWithStyle: rowAction.contextualActionStyle
+                                                                                      title: rowAction.title handler:^(UIContextualAction * _Nonnull action, __kindof UIView * _Nonnull sourceView, void (^ _Nonnull completionHandler)(BOOL)) {
+            VMKViewModel *viewModel = [rowAction swipedRowAction];
+            BOOL isRowModified = NO;
+            if (viewModel) {
+                // positions were voided
+                VMKTableViewDataSource *tableViewDataSource = self.tableView.dataSource;
+                BOOL shouldOverrideSourceViewForSwipeAction = [rowAction.delegate shouldOverrideSourceViewForSwipeAction];
+                UIView * cellView = [self getSuperViewOfClass: swipeContainer fromView: sourceView];
+                //UIView * cellView = sourceView.superview.superview;
```

Comment:
I think keep this for now and once we have some better approach then replace with that @tovnixon

Reply:
rolled back